tabo-hdinsight/Azuredeploy.json

```diff
@@ -0,0 +1,555 @@
+      "type": "Microsoft.Compute/virtualMachines",
+      "apiVersion": "2015-06-15",
+      "location": "[variables('TaboLocation')]",
+      "dependsOn": [
+        "[concat('Microsoft.Network/networkInterfaces/',variables('TaboNetworkInterfaceName'))]",
+        "[concat('Microsoft.Storage/storageAccounts/',variables('TaboStorageAccountName'))]"
+      ],
+      "properties": {
+        "osProfile": {
+          "computerName": "[variables('TaboVirtualMachineName')]",
+          "adminUsername": "[variables('TaboAdminUsername')]",
+          "adminPassword": "[variables('TaboAdminPassword')]",
+          "windowsConfiguration": {
+            "provisionVmAgent": "true"
+          }
+
+        },
+        "hardwareProfile": {
+          "vmSize": "[variables('TaboVirtualMachineSize')]"
+        },
+        "storageProfile": {
+          "imageReference": {
+            "publisher": "MicrosoftSQLServer",
+            "offer": "SQL2016-WS2012R2",
+            "sku": "Enterprise",
+            "version": "latest"
+          },
+          "osDisk": {
+            "name": "[variables('TaboVirtualMachineName')]",
+            "vhd": {
+              "uri": "[concat(concat(reference(resourceId('Microsoft.Storage/storageAccounts', variables('TaboStorageAccountName')), '2015-06-15').primaryEndpoints['blob'], 'vhds/'), variables('TaboVirtualMachineName'), '20168117238.vhd')]"
+            },
+            "createOption": "fromImage"
+          },
+          "dataDisks": [
+            {
+              "name": "[concat(variables('TaboVirtualMachineName'), '-disk-1')]",
+              "vhd": {
+                "uri": "[concat(concat(reference(resourceId('Microsoft.Storage/storageAccounts', variables('TaboStorageAccountName')), '2015-06-15').primaryEndpoints['blob'], 'vhds/'), variables('TaboVirtualMachineName'), '-disk-1-20168117238', '.vhd')]"
+              },
+              "createOption": "empty",
+              "lun": 0,
+              "diskSizeGB": "1023",
+              "caching": "ReadOnly"
+            }
+          ]
+        },
+        "networkProfile": {
+          "networkInterfaces": [
+            {
+              "id": "[resourceId('Microsoft.Network/networkInterfaces', variables('TaboNetworkInterfaceName'))]"
+            }
+          ]
+        }
+
+      },
+      "resources": [
+        {
+          "name": "cse",
+          "type": "extensions",
+          "location": "[resourceGroup().location]",
+          "apiVersion": "2015-06-15",
+          "dependsOn": [
+            "[variables('TaboVirtualMachineName')]"
+          ],
+          "tags": {
+            "displayName": "cse"
+          },
+          "properties": {
+            "publisher": "Microsoft.Compute",
+            "type": "CustomScriptExtension",
+            "typeHandlerVersion": "1.8",
+            "autoUpgradeMinorVersion": true,
+            "settings": {
+              "fileUris": [ "https://sapiotmlblob.blob.core.windows.net/hdinsightscript/sqlnew.ps1" ],
```

Comment:
Please see #2 in best practice checklist: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md